keyboards/ibm/model_m/modelh/keyboard.json

```diff
@@ -35,7 +38,7 @@
         "device_version": "1.0.0",
         "max_power": 100,
         "pid": "0xB155",
-        "vid": "0xFEED"
+        "vid": "0x1D50"
```

Comment:
The `0x1D50` VID is now supposed to be managed through https://github.com/openmoko/openmoko-usb-oui, and there is no `0xB155` PID registered there.  If you want to use that VID, please register the PID there properly first.

Reply:
Chose new random VID